### ISSUE TITLE
docs: 保研指南/基本概念 添加保研资格章节

### ISSUE THREAD

**Issue by Axi404** [CS-BAOYAN/CS-BAOYAN-Wiki#61]
保研资格章节介绍保研资格的获取，也就是综测分排名。
注明，综测分除了在保研资格的时候有用，在大三期间主要起到作用的是智育分以及对应的排名。

**Comment by ZhikangNiu** [comment]
我觉得这个点其实每个学校的政策不一样（所以关于保研资格可以举几个例子就ok

**Comment by Justjustifyjudge** [comment]
> 我觉得这个点其实每个学校的政策不一样（所以关于保研资格可以举几个例子就ok

exactly, 比如SCUT的保研资格既不是综测，也不是绩点，而是绩点+科研分+体育分+竞赛分。但是科研/体育/竞赛分的获取方式是独立指定的保研加分标准文件，而非参考综测。
